### PR TITLE
Prepare v12.0.0 release

### PR DESCRIPTION
**Pull request by barmac**
This updates changelog to v12.0.0.

### CODE REVIEW

File: packages/dmn-js-drd/test/spec/features/auto-place/DmnAutoPlaceSpec.js
Comment (pinussilvestrus) [inline packages/dmn-js-drd/test/spec/features/auto-place/DmnAutoPlaceSpec.js:9]: Sorry if I don't have enough context here, but is this intended?

Comment (pinussilvestrus) [reply packages/dmn-js-drd/test/spec/features/auto-place/DmnAutoPlaceSpec.js:9]: Ah sorry, I see: https://github.com/bpmn-io/dmn-js/pull/684/commits/f26351a988ea9d89d618fb9540cf83d275972b2f

Interesting though!

**Review by pinussilvestrus — COMMENTED** [review]
For me it looks good, waiting for approval from another reviewer (@MaxTru) who has more insights about the recent changes.

File: packages/dmn-js-drd/test/spec/features/auto-place/DmnAutoPlaceSpec.js
Comment (barmac) [reply packages/dmn-js-drd/test/spec/features/auto-place/DmnAutoPlaceSpec.js:9]: I hope that this will improve the coverage metrics a bit, but it's probably not the only problem with the coverage in this project.

**Review by MaxTru — APPROVED** [review]
:tada:

**Pull request merged**
